src/Application/Regulation/Command/SaveRegulationLocationCommandHandler.php

```diff
@@ -45,6 +48,20 @@ public function __invoke(SaveRegulationLocationCommand $command): Location
             foreach ($command->measures as $measureCommand) {
                 $measureCommand->location = $location;
                 $measure = $this->commandBus->handle($measureCommand);
+
+                $locationNew = $this->locationNewRepository->add(
+                    new LocationNew(
+                        uuid: $this->idFactory->make(),
+                        measure: $measure,
+                        cityLabel: $command->cityLabel,
+                        cityCode: $command->cityCode,
+                        roadName: $command->roadName,
+                        fromHouseNumber: $command->fromHouseNumber,
+                        toHouseNumber: $command->toHouseNumber,
+                        geometry: $geometry,
+                    ),
+                );
+                $measure->addLocation($locationNew);
                 $location->addMeasure($measure);
```

Comment:
(Détail) Déplacer la ligne `$location->addMeasure($measure)` pour regrouper ce qui concerne l'ancienne location et la nouvelle

```suggestion
                $measure = $this->commandBus->handle($measureCommand);
                $location->addMeasure($measure);

                $locationNew = $this->locationNewRepository->add(
                    new LocationNew(
                        uuid: $this->idFactory->make(),
                        measure: $measure,
                        cityLabel: $command->cityLabel,
                        cityCode: $command->cityCode,
                        roadName: $command->roadName,
                        fromHouseNumber: $command->fromHouseNumber,
                        toHouseNumber: $command->toHouseNumber,
                        geometry: $geometry,
                    ),
                );
                $measure->addLocation($locationNew);
```